taqueria-plugin-flextesa/proxy.ts

```diff
@@ -545,19 +521,22 @@ const importSandboxAccounts = (parsedArgs: ValidOpts) =>
 
 const addSandboxAccounts = (parsedArgs: ValidOpts) => maybeInstantiateAccounts(parsedArgs);
 
-const monitor = (parsedArgs: ValidOpts): Promise<void> => {
-	const sandbox = getSandbox(parsedArgs);
-	if (sandbox) {
-		return getContainerName(parsedArgs)
-			.then(container => spawnCmd(`docker exec ${container} octez-client rpc get /monitor/valid_blocks`))
-			.then(() => {});
-	} else return sendAsyncErr('Please specify the name of a sandbox to monitor.');
-};
-
 const getDefaultSandboxName = (parsedArgs: Opts) => {
 	return Object.keys(parsedArgs.config.sandbox ?? {}).shift() ?? 'local';
 };
 
+const taskMap: Record<string, (opts: ValidOpts) => Promise<void>> = {
+	'list accounts': listAccountsTask,
+	'show protocols': listProtocolsTask,
+	'list protocols': listProtocolsTask,
+	'start sandbox': startSandboxTask,
+	'start flextesa': startSandboxTask,
```

Comment:
I thought `yargs` ignores the second word so `start sandbox` and `start flextesa` and `start <anything>` are the same no? When did we fix that?

Reply:
`start flextesa` has always been an alias for the `start sandbox` function, and thus, it was technically correct to include them here. 

Also, we don't use `start [words...]` for any other task than `start sandbox`, and thus, the alias will actually work as yargs appears to do the following:

- `start sandbox` is registered as a command in yargs
- You can then type `start [anything]` and yargs will match that against the `start sandbox` task. Thus, `start flextesa` will get matched against `start sandbox`, which is correct! :P

Reply:
Yes but this is quite bad because the second word is in fact meaningless. Since it doesn't cause problem in this particular case, I am ok with moving on with this. We do need to fix this sometime in the future though.